### DISCUSSION TITLE
Filament runout sensor

### DISCUSSION THREAD

**Discussion by heidepiek** [The-300lb-Gorilla/Ender-2-Pro-v427#6] [Show and tell]
Printed a filament runout mount sensor for our ender 2 pro. I saw you could enable it in the screen. So why not! Coincidentally, the runout sensor is also set correctly in the firmware. 

https://user-images.githubusercontent.com/104028471/229298066-b495df9b-71b3-475c-8ebd-d8f514ef488f.mp4

Greetzzz from holland

**Comment by The-300lb-Gorilla** [comment]
In the firmware, the default is set to "LOW". It is only set to "HIGH" if
the "+" and "-" are reversed. Glad to hear you are making it your own
version of the Ender 2 Pro.

**Reply by heidepiek** [reply]
Am not a profesional, but have put a video on youtube. Hope you get some more followers on your repository.

https://www.youtube.com/watch?v=qAja13pnw5g

Greetzz harrie
